src/main/webapp/app/entities/project/project.service.ts

```diff
@@ -45,6 +48,14 @@ export class ProjectService {
       .pipe(map((res: EntityArrayResponseType) => this.convertDateArrayFromServer(res)));
   }
 
+  responsibilities(project: Project, req?: any): Observable<ResponsibilityArrayResponseType> {
```

Comment:
würde hier ein $ erwarten, da es ein Observable zurück gibt. Können wir aber auch nicht so stark forcieren und es so belassen. Bin einfach ein Fan von Konsistenz ;)

Reply:
Ah und etwas viel wichtigeres: Gibt es nicht ein eigener Service für die responsibilities?

Reply:
bei Methoden würde ich kein `$` machen. alle Service-Methoden geben wohl eine `Observable` zurück..?
den `Responsibility`-Service gibt es, das habe ich mit Sven diskutiert. jedoch geht es hier ja darum, die responsibilities pro projekt abzufragen. die url ist deshalb auch `/project/:id/responsibilities` (backend).

Reply:
Ja aber das ist ja egal, ob es responsibilities eines Projektes sind oder nicht. Die Methode gibt ja responsibilities zurück was meiner Meinung nach dann auch in den entsprechenden service gehört

Reply:
man könnte es verschieben und responsibilitiesByProjectId$ nennen. Braucht ja nicht das ganze projekt.

Und wegen dem Dollar: Ein Service kann ja eigentlich irgendetwas machen. Deshalb fände ich es überall gut, wo man ein Observable hat. Aber ist Geschmackssache. Können es gut so belassen

Reply:
naja, aber diese `Responsibilities` gehören eben fachlich zu einem `Project`. ich werds verschieben, es macht eben beides irgendwie Sinn. dieser Satz hat mich überzeugt:
> Die Methode gibt ja responsibilities zurück was meiner Meinung nach dann auch in den entsprechenden service gehört


`$` für Observables find ich ok - in JS. aber wir benutzen TypeScript? xD es ist typisiert und wir haben alle gute IDEs. den Punk sehe ich wirklich nicht.. ist in etwa wie `m_` in Java für Member-Variablen.
dann müssten alle bestehenden Methoden durchsucht und angepasst werden. ich bin dafür, dass in einem Issue als irgendwann-Feature-Feature festzuhalten. wenn du dann einen Samstag aufwenden willst, von mir aus 🤪